website/content/partials/cli/monitor/flags/log-format.mdx

```diff
@@ -0,0 +1,9 @@
+<a id="monitor-flag-log-format" />
+
+**`-log-format (string: "standard")`**
+
+Format to emit logs.
+Valid formats are "standard", and "json".
+If this option is not specified, "standard" is used.
+
+**Example**: `-log-format="standard"`
```

Comment:
```suggestion
**Example**: `-log-format "standard"`
```
Technically flags don't need the `=`. It doesn't hurt to include it (the CLI won't complain), but not including it is another visual indicator that this is a flag rather than an option (which does need the `=`)